internal/dataexchange/dxfactory/factory.go

```diff
@@ -44,7 +44,7 @@ func InitConfig(config config.ArraySection) {
 }
 
 func InitConfigDeprecated(config config.Section) {
-	config.AddKnownKey(coreconfig.PluginConfigType, NewFFDXPluginName)
+	config.AddKnownKey(coreconfig.PluginConfigType)
```

Comment:
Yea, I tagged this PR with "migration consideration" in the hopes that one of us remembers why. Thankfully the CLI has explicitly populated "type" for a while (since https://github.com/hyperledger/firefly-cli/pull/156). However, we will want to cut a 1.0.x patch release that includes https://github.com/hyperledger/firefly/pull/882, and recommend upgrading to that and fixing all config warnings as part of the migration journey to 1.1.